src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/ProbabilityTable/Builder/src/interpolationParameter.hpp

```diff
@@ -0,0 +1,11 @@
+Builder& interpolationParameter( int iP ){
+  if( ( iP != 2 ) and ( iP != 5 ) ){
+    Log::error( "Invald interpolation parameter: {}", iP );
+    Log::info( "Interpolation parameter between unresolved resonance tables "
+               " must be 2 or 5" );
+    throw std::range_error( "Invalid interpolation parameter" );
```

Comment:
I would not use range_error in this case.

Reply:
I'm open to other suggestions. If you don't like `std::range_error`, what would be better?